src/api/v1/teams/index.js

```diff
@@ -0,0 +1,40 @@
+import { Router } from 'express'
+import validations from './team_validation'
+import { validate, validateTeamId } from '../../../middlewares/validation'
+import { createTeam, getAllTeams, getATeam, deleteTeam, addUsersToTeam } from './team_controllers'
+import { isAdmin, isAuthenticated } from '../../../middlewares/auth'
+
+const router = Router()
+
+/**
+ * team creation by admin
+ * /api/v1/teams/new - POST
+ */
+router.post('/teams/new', validations.teamCreateRules(), validate, createTeam)
```

Comment:
Let's stick to one endpoint for POST and GET methods
```suggestion
router.post('/teams', validations.teamCreateRules(), validate, createTeam)
```